src/argus/notificationprofile/media/sms_as_email.py

```diff
@@ -79,8 +80,14 @@ def has_duplicate(self, queryset: QuerySet, settings: dict) -> bool:
         """
         return queryset.filter(settings__phone_number=settings["phone_number"]).exists()
 
-    @staticmethod
-    def send(event: Event, destinations: QuerySet[DestinationConfig], **_) -> bool:
+    @classmethod
+    def get_relevant_addresses(cls, destinations: List(DestinationConfig)) -> List:
```

Comment:
Again, returns a List of what? Also:
```suggestion
    def get_relevant_addresses(cls, destinations: List[DestinationConfig]) -> List:
```